libs/components/ag-grid/src/lib/modules/ag-grid/types/cell-editor-utils.ts

```diff
@@ -30,6 +30,9 @@ export class SkyAgGridCellEditorUtils {
     }
   }
 
+  /**
+   * Returns the difference between the minuend and subtrahend, or 0 if the minuend is not defined.
```

Comment:
```suggestion
   * Returns the difference between the minuend and subtrahend. If the minuend is not defined, returns 0.
```
Also, do we need to define "minuend" and "subtrahend"? I don't know what either of these are, and if we don't expect consumers to intuitively understand what they are, then I don't think this description is very useful.